cdap-tms/pom.xml

```diff
@@ -176,6 +176,12 @@
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.cdap.cdap</groupId>
+      <artifactId>cdap-messaging-spi</artifactId>
+      <version>${project.version}</version>
+      <scope>compile</scope>
```

Comment:
compile is default, so no need to specify it (unless it's overriden for whatever reason in dependency management)